config/notifiers.go

```diff
@@ -283,3 +296,35 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "opsgenie config")
 }
+
+type PushoverConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	UserKey  Secret `yaml:"user_key"`
+	Token    Secret `yaml:"token"`
+	Title    string `yaml:"title"`
+	Message  string `yaml:"message"`
+	URL      string `yaml:"url"`
+	Priority string `yaml:"priority"`
+	Retry    int    `yaml:"retry_seconds"`
```

Comment:
I know you just changed that as asked by @brian-brazil, but why are we not using golangs `time.Duration` here? `retry: 1m`, `expire: 1h`, maye with an added `interval` suffix or so.